pkg/gadgets/traceloop/annotations/annotations.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019-2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package annotations
```

Comment:
```suggestion
package labels
```

It is using labels and not annotations:
```
$ kubectl get trace -n gadget -o yaml
apiVersion: v1
items:
- apiVersion: gadget.kinvolk.io/v1alpha1
  kind: Trace
  metadata:
    ...
    labels:
      ...
      type: global
    name: traceloop-jttjf
    namespace: gadget
```